app/components/billing/account.js

```diff
@@ -31,6 +34,12 @@ export default Component.extend({
 
   showPlansSelector: false,
   showAddonsSelector: false,
+  isTrialProcessCompleted: computed(function () {
+    return !this.isTrial;
+  }),
+  isEduProcessCompleted: computed(function () {
+    return !this.isEducation;
+  }),
```

Comment:
Shouldn't these properties track changes on dependent ones? Or are they just initialization values?
```suggestion
  isTrialProcessCompleted: not('isTrial'),
  isEduProcessCompleted: not('isEducation'),
```

Reply:
They are just initialisation values, that's correct!

Reply:
Thanks, I thought it would be the case 👍